docs/README.md

```diff
@@ -77,6 +79,7 @@ If you have been familiar with SkyWalking, you could use this catalog to find th
       * [Dynamic Configuration](en/setup/backend/dynamic-config.md). Make configuration of OAP changed dynamic, from remote service or 3rd party configuration management system.
       * [Uninstrumented Gateways](en/setup/backend/uninstrumented-gateways.md). Configure gateways/proxies that are not supported by SkyWalking agent plugins, to reflect the delegation in topology graph.
     * [UI setup document](en/setup/backend/ui-setup.md).
+    * [CLI setup document](https://github.com/apache/skywalking-cli).
```

Comment:
All these two are just menu, you should add this into the document content directly. People could read the document through the main entance, especially for the new user.

Such as https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/concepts-and-designs/README.md

Reply:
Including this
![image](https://user-images.githubusercontent.com/5441976/70526593-e469e680-1b84-11ea-9a06-32d746ef357d.png)

and  https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/setup/backend/backend-ui-setup.md

Reply:
Another reminder, we don't have a download link from the CLI repo, https://github.com/apache/skywalking-cli.

Reply:
> Another reminder, we don't have a download link from the CLI repo, https://github.com/apache/skywalking-cli.

adding

Reply:
Still missing the update on https://github.com/apache/skywalking/blob/898074e7784ab89b54b896be28d947f8fafbd489/docs/en/setup/backend/backend-ui-setup.md, I think.